wgengine/netstack/netstack_userping_apple.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build darwin || ios
+
+package netstack
+
+import (
+	"github.com/prometheus-community/pro-bing"
```

Comment:
no goimports formatting on this? normally non-std imports go in their own section after a blank line

so:

```go
import (
   "net/netip"
    "time"

    "github.com/...."
)
```

Reply:
fixed.